package.json

```diff
@@ -74,6 +74,12 @@
     "@commitlint/cli": "13.1.0",
     "@commitlint/config-conventional": "13.1.0",
     "@openapitools/openapi-generator-cli": "2.4.14",
+    "@lerna-lite/core": "1.4.0",
```

Comment:
hi there, Lerna-Lite maintainer here... you don't actually need to have `core`, neither `publish` since these 2 are already dependencies of `@lerna-lite/cli`, see Lerna-Lite [installation](https://github.com/ghiscoding/lerna-lite#installation) table. To be clear, you could also leave them there but it's just more dependencies that you'll have to keep updating, so up to you 😉 However `exec`, `list` and `run` must be kept because those are optional and not included in the CLI. Cheers and Thanks for trying Lerna-Lite

Reply:
Thanks a lot @ghiscoding for the insights. :)

Reply:
thank you @ghiscoding will try and test it!